modules/util/util.go

```diff
@@ -230,6 +230,20 @@ func IfZero[T comparable](v, def T) T {
 	return v
 }
 
+// DefArg helps the "optional argument" in Golang: func(foo string, optionalArg ...int)
+// it returns the first non-zero value from the given optional argument,
+// or the default value if there is no optional argument.
+func DefArg[T any](defArgs []T, def T) (ret T) {
```

Comment:
Neat! Can you rename "Def" to "Default" for this function and variables? I think it'll make it more intuitive when reading code referencing this function.

Reply:
Thanks, done in ea9e0aebccc72ab0c44b76b3f9ade0a1e3a99b6d

`Default` is too general .... indeed it is for function argument, so does `DefaultArg` look good to you?